src/assembler/encoder.ts

```diff
@@ -340,7 +340,9 @@ function writeDataInstruction(
     case 'SPACE':
     case 'FILL':
     case '%':
-      bytes = Array(values[0]).fill(Byte.fromUnsignedInteger(0x00))
+      bytes = Array(evaluateExpression(instruction)).fill(
+        Byte.fromUnsignedInteger(0xff)
```

Comment:
is fill intentionally with ff instead of 0?

Reply:
Fixed this